.idea/compiler.xml

```diff
@@ -7,6 +7,7 @@
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
         <module name="JDosAttacker" />
+        <module name="JDosAttacker-" />
```

Comment:
I really don't understand why I added `-` to the module name, try removing it if I can

Reply:
Okay